pkg/gen/nodejs-templates/helm.ts.mustache

```diff
@@ -188,7 +199,7 @@ export namespace v2 {
     // that must be created first, are.
     //
     // [1]: https://github.com/helm/helm/blob/094b97ab5d7e2f6eda6d0ab0f2ede9cf578c003c/pkg/tiller/kind_sorter.go
-    /* @internal */ export function helmSort(a: { kind: string }, b: { kind: string }): number {
+    /** @ignore */ export function helmSort(a: { kind: string }, b: { kind: string }): number {
```

Comment:
It was showing up in the API docs: https://pulumi.io/reference/pkg/nodejs/@pulumi/kubernetes/v2/#helmSort

Not sure if that was related to the `/*` rather than `/**`, but `@ignore` or `@hidden` were the keywords I saw in the `typedoc` documentation.

Reply:
@CyrusNajmabadi, thoughts?

Reply:
it probably needed to be `/** @internal */` not `/* @internal */`.  i.e. two `**`s.